database/migrations/2021_06_12_161530_create_attendances_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateAttendancesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('attendances', function (Blueprint $table) {
```

Comment:
You will need to add foreign id fields for the user_id and course_id otherwise we cant be able to link the two in the future.